app/labor/rate_limit_checker.rb

```diff
@@ -4,9 +4,18 @@ class RateLimitChecker
   RETRY_AFTER = {
     article_update: 30,
     image_upload: 30,
-    published_article_creation: 30
+    published_article_creation: 30,
+    organization_creation: 300
   }.with_indifferent_access.freeze
```

Comment:
Can we include a comment to indicate that these are all in seconds? Initially I thought that these were the number of attempts and had to look through the rest of the file to verify that I was wrong and it was actually seconds.